cmd/topology-aware/nri-resmgr-topology-aware-deployment.yaml.in

```diff
@@ -117,20 +117,32 @@ spec:
         hostPath:
           path: /var/lib/nri-resmgr
           type: DirectoryOrCreate
-      - name: nri-resmgr
         hostPath:
           path: /var/lib/nri-resmgr
       - name: hostsysfs
         hostPath:
           path: /host
           type: DirectoryOrCreate
-      - name: sys
         hostPath:
           path: /sys
           type: Directory
       - name: resmgrsockets
         hostPath:
           path: /var/run/nri-resmgr
```

Comment:
Is this (removing the `name`s) intentional ? Doesn't that leave us with a single entry that has several `hostPath`s of which only the last one being effective ?

Reply:
@klihub Exactly, I found this issue while using kustomize that we have two hostPath entries under the same name.
Do we need only /host or /sys?

We also have duplication for /var/run/nri-resmgr which should be fixed in https://github.com/jukkar/nri-resmgr/pull/16

Reply:
`/sys` should be added by the runtime by default but that's not the full sysfs. We'd need the full sysfs bind-mounted to `/host/sys` so we can do HW-discovery. For TA, ro-mounting it should be enough. For balloons we nee rw-mount for fiddling around with CPU frequency scaling max's.